website/docs/rollbacks.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Partially Failed Commits
```

Comment:
Propose a title like "Rollback nechansim" since this page is organized under table service. And move the use case of partially failed commits into a background/intro paragraph. Additionally we can also add a FAQ to connect the use case with this page to help discoverability.